custom_components/tryfi/pytryfi/common/query.py

```diff
@@ -104,6 +104,45 @@ def getDevicedetails(session: requests.Session, petId: str):
     LOGGER.debug(f"getDevicedetails: {response}")
     return response['data']['pet']
 
+def getPetHealthTrends(session: requests.Session, petId: str, period: str = 'DAY'):
+    """Get pet health trends including behavior data for Series 3+ collars."""
+    # Build the query with the health trends fragment inline
+    qString = f"""
+    query PetHealthTrends {{
+        getPetHealthTrendsForPet(petId: "{petId}", period: {period}) {{
+            __typename
+            period
+            behaviorTrends {{
+                __typename
+                id
+                title
+                summaryComponents {{
+                    __typename
+                    eventsSummary
+                    durationSummary
+                }}
+                chart {{
```

Comment:
Are we using the historical chart data in the periodic refresh? If not, then I'd suggest removing it to reduce the size of the GraphQL since it looks like a time series. Then to fetch the history, add another GraphQL call just for that. This helps keep our API response payloads small and performant.